src/site/content/en/blog/web-share/index.md

```diff
@@ -47,8 +47,9 @@ Web share has the following capabilities and limitations:
 * It must be invoked in response to a user action such as a click. Invoking it
   through the `onload` handler is impossible.
 * It can share, URLs, text, or files.
-* As of mid 2020, it is only available on Safari and on Android in Chromium
-  forks. See
+* As of January 2021, it is available on Android in Chromium forks, Chrome OS,
+  and Chrome for Windows. Chrome for Safari on MacOS is still in development.
```

Comment:
I assume you meant `Chrome on macOS` and `Chrome on Windows`. Not safari ;)

Reply:
It's apparently still in development on MacOS. See the [status entry comments](https://www.chromestatus.com/feature/5668769141620736).

Reply:
I meant "Chrome on Windows", not "Chrome for Windows".
And not "Chrome for Safari on macOS", but just "Chrome on macOS" as Safari is another browser from another company ;)

Reply:
I'm having trouble getting this in my head. Can you do this as a suggestion, please.

Reply:
Here's what I would write.
```
* As of January 2021, it is available on Safari, on Android in Chromium forks,
  on Chrome OS, and Chrome on Windows. Chrome on MacOS is still in development.
  ```

Reply:
Done. Please take another look.